fplus-lib/src/core/mod.rs

```diff
@@ -1759,13 +1759,7 @@ impl LDNApplication {
             .await
             .map_err(|e| LDNError::Load(format!("here1 {}", e)))?;
         let app = match ApplicationFile::from_str(&file) {
-            Ok(app) => {
-                if app.lifecycle.is_active {
-                    app
-                } else {
-                    return Ok(None);
-                }
-            }
+            Ok(app) => app,
```

Comment:
FE needs to be ready before we deploy this. By default it should still only show open applications, and show closed applications when some checkbox is checked.